packages/components/src/components/SquiggleViewer/VariableBox.tsx

```diff
@@ -101,57 +132,95 @@ export const VariableBox: FC<VariableBoxProps> = ({
     };
   }, []);
 
+  const isCollapsed = isFocused ? false : settings.collapsed;
+  const shouldShowTriangleToggle = !isFocused;
+  const shouldShowLeftHeaderPreview = !isFocused && !!preview;
+  const shouldShowRightHeaderItems = !settings.collapsed || isFocused;
+  const shouldShowRightHeaderString = Boolean(
+    shouldShowRightHeaderItems && path.items.length
+  );
+  const shouldShowRightHeaderFindInEditor = !isRoot;
+  const showOpenedLeftSidebar = Boolean(
+    !isCollapsed && path.items.length && !isFocused
+  );
+  const _focus = () => !isFocused && path.items.length && focus(path);
+
   return (
     <div ref={saveRef}>
       {name === undefined ? null : (
-        <header className="flex justify-between hover:bg-stone-100 rounded-md">
+        <header
+          className={clsx(
+            "flex justify-between group",
+            isFocused ? "mb-2" : "hover:bg-stone-100 rounded-md"
+          )}
+        >
           <div className="inline-flex items-center">
+            {shouldShowTriangleToggle && (
+              <span
```

Comment:
Minor note: `<span>...</span><div>...</div>` doesn't make much sense semantically, so I usually just use `<div>` for flexbox children.

Reply:
Good point